src/main/java/org/spectra/cluster/tools/CliOptions.java

```diff
@@ -0,0 +1,231 @@
+package org.spectra.cluster.tools;
+
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.OptionBuilder;
+import org.apache.commons.cli.Options;
+
+/**
+ * Created with IntelliJ IDEA.
+ * User: jg
+ * Date: 9/15/13
+ * Time: 11:35 AM
+
+ */
+public class CliOptions {
+    public enum OPTIONS {
+
+        CONFIG_FILE("config", "c"),
+        OUTPUT_PATH("output.path", "o"),
+
+        INPUT_FILES("input.files", "i"),
+
+        PRECURSOR_TOLERANCE("precursor.tolerance", "p"),
```

Comment:
For backwards compatibility reasons it might be better to stick to the "_" as a separator. What do you think?
```suggestion
        PRECURSOR_TOLERANCE("precursor_tolerance", "p"),
```

Reply:
I don't really mind. The "." structure is well known is Java for parameters name convention. Here are the conversions https://commons.apache.org/proper/commons-cli/  ... I like the dot style but it could be precursor-tolerance. I haven't seen _ . If we don't have a long community would be nice to keep a standard. By I don't have a strong feeling.

Reply:
I also agree that we can easily change this at this stage. I wasn't aware that this was the convention for Java parameter names. In UNIX tools the "-" is more common. But that wasn't supported in the commons-cli package when I created the first CLI. So I'm happy with whatever you prefer.

Reply:
Lets go for the "." approach and we can change easily in the future.